src/components/routes/Host/ListingForm/ListingForm.tsx

```diff
@@ -60,9 +60,9 @@ interface Props extends RouterProps {
 }
 
 const ListingFormSchema = Yup.object().shape({
-  addressLine1: Yup.string().min(1, 'Too Short!'),
+  addressLine1: Yup.string().required('Address Line 1 is a required field'),
```

Comment:
can we move these error messages to a utils folder for easy access/altering?

Reply:
just made a `formatError` function in `utils/formatter`. apollo gql also have a `formatError` function; hope the imports won't be confusing with the same function name